samples/highcharts/demo/pie-chart/demo.js

```diff
@@ -1,6 +1,14 @@
 Highcharts.chart('container', {
     chart: {
-        type: 'pie'
+        type: 'pie',
+        zooming: {
+            type: 'xy'
+        },
+        panning: {
+            enabled: true,
+            type: 'xy'
+        },
+        panKey: 'shift'
```

Comment:
Zoom select rectangle has weird sizing effect along y-axis. Not sure, if this is by design, but it makes UX strange when first moving downwards than to the right side.

Reply:
Maybe it is also because the tooltip covers the little rectangle. Can we disable tooltip while setting zoom range?